packages/app/test/cypress/integration/30-search/search.spec.ts

```diff
@@ -68,3 +68,138 @@ context('Access to legacy private pages', () => {
   });
 
 });
+
+context('Search all pages', () => {
+  const ssPrefix = 'search-all-pages-';
+
+  beforeEach(() => {
+    // login
+    cy.fixture("user-admin.json").then(user => {
+      cy.login(user.username, user.password);
+    });
+    // collapse sidebar
+    cy.collapseSidebar(true);
+  });
+
+  it(`Search all pages by word is successfully loaded`, () => {
+    const searchText = 'help';
+    const tag = 'help'
+    cy.visit('/Sandbox');
+    // Add tag
+    cy.get('#edit-tags-btn-wrapper-for-tooltip > a').click({force: true});
+    cy.get('#edit-tag-modal').should('be.visible');
+
+    cy.get('#edit-tag-modal').within(() => {
+      cy.get('.rbt-input-main').type(tag).trigger('change');
+      cy.get('#tag-typeahead-asynctypeahead').should('be.visible');
+      cy.get('#tag-typeahead-asynctypeahead-item-0').should('be.visible');
+      cy.get('a#tag-typeahead-asynctypeahead-item-0').click({force: true})
+    });
+
+    cy.get('#edit-tag-modal').within(() => {
+      cy.get('div.modal-footer > button').click();
+    });
```

Comment:
This code (Adding a tag) do not work?

Reply:
Yes this lines doesn't work in live test but in my local it works , because tag `help` is exists in the database 
## Live 
![image](https://user-images.githubusercontent.com/92426728/175875910-08c127eb-e02c-4f4a-be70-61c58f4bba4f.png)

## Local
![TEST](https://user-images.githubusercontent.com/92426728/175875938-3a199fa6-8508-4b91-a063-cc6e23d4d64e.png)

## Test from [GW7740 ](https://youtrack.weseek.co.jp/issue/GW-7740)
![tag-operations-add-new-tag-type-tag-name](https://user-images.githubusercontent.com/92426728/175876349-c013a85b-6a2e-46ef-b340-1d64944ecec5.png)
It still working